tests/run-pass/fs.rs

```diff
@@ -1,8 +1,13 @@
 // ignore-windows: File handling is not implemented yet
 // compile-flags: -Zmiri-disable-isolation
 
+#![feature(rustc_private)]
+
+extern crate libc;
```

Comment:
This test otherwise tests the `std::fs` module, I'd rather not confuse that with testing lower-level APIs.

Maybe create a new file `run-pass/libc.rs` and add this test there? We'll probably gain more tests like that over time.